base-template.sh

```diff
@@ -138,8 +138,8 @@ execute_lfs_hook_if_appropriate() {
     if [ "$GIT_LFS_AVAILABLE" = "true" ]; then
         git lfs "$HOOK_NAME" "$@" || return 1
     elif [ "$REQUIRES_LFS_SUPPORT" = "true" ]; then
-        echo "This repository requires Git LFS, but \`git-lfs\` was not found on your PATH."
-        echo "If you no longer want to use Git LFS, remove the \`.githooks/.lfs-required\` file."
+        echo "! This repository requires Git LFS, but \`git-lfs\` was not found on your PATH." >&2
+        echo "!  If you no longer want to use Git LFS, remove the \`.githooks/.lfs-required\` file." >&2
```

Comment:
As a general theme:
- please keep the `!` on the start of the first line only (no need to have all lines start with `!`)
- please don't change the indentation on lines, they should start where the previous did in most cases
Example:

```
! This is the first line of error
  This is some text explaining what to do, etc.
```

Reply:
Ah I wasnt sure, so now its clear, I though you wanted to distinguise further error message from each other by indenting...

Reply:
So one "!" marks one error! I correct that overall